app/views/registration/CurrentTaxYear.scala.html

```diff
@@ -87,11 +84,6 @@ <h1 id="title" class="govuk-heading-xl">
                                     <input type="hidden" id='actives_@{index}_uid' name='actives[@index].uid' value='@activeField("uid").value.get'/>
                                     <input class="govuk-checkboxes__input" type="checkbox" id='checkbox-@activeField("uid").value.get' name='actives[@index].active' value="true" @if(activeField("uid").value.get == "47") {aria-describedby="actives[@index].hint"}/>
                                     <label class="govuk-label govuk-checkboxes__label" for="checkbox-@activeField("uid").value.get">@messages("BenefitInKind.label." + activeField("uid").value.get)</label>
-                                    @if(activeField("uid").value.get == "47") {
```

Comment:
That if has been removed, because upper if already controlling that with `!=` and not possible to come here with `==`, Upper if:
```
@if(activeField("uid").value.isDefined && activeField("uid").value.get != "47"
```